src/core/Button/__snapshots__/Button.test.tsx.snap

```diff
@@ -68,11 +68,12 @@ exports[`calling render with the same component on the same container does not r
   font-size: 16px;
   line-height: 1.5;
   font-weight: 600;
-  line-height: 1;
-  padding: 8px 16px;
+  font-size: 14px;
```

Comment:
This needs a new typography token before merge.

Reply:
This typography definition will be replaced with a token later on. Design inventory and definition work needs to be done before adding the tokens and this would be blocking these changes for too long.